docs/code/expressions.md

```diff
@@ -5,31 +5,26 @@ contentType: howto
 
 # Expressions
 
---8<-- "_snippets/code/tournament-notes.md"
+Expressions are a powerful feature implemented in all n8n nodes. They allow node parameters to be set dynamically based on data from:
 
-Use [expressions](/glossary.md#expression-n8n) to set node parameters dynamically based on data from:
-
-- Previous nodes
+- Previous node executions
 - The workflow
 - Your n8n environment
 
-You can execute JavaScript within an expression. 
+You can also execute JavaScript within an expression, making this a convenient and easy way to manipulate data fields into useful parameter values without having to resort to writing extensive extra code.
 
-n8n created and uses a templating language called [Tournament](https://github.com/n8n-io/tournament){:target=_blank .external-link}, and extends it with [custom methods and variables](/code/builtin/overview.md) and [data transformation functions](/code/builtin/data-transformation-functions/index.md) that help with common tasks, such as retrieving data from other nodes, or accessing metadata.
+n8n created and uses a templating language called [Tournament](https://github.com/n8n-io/tournament){:target=_blank .external-link}, and extends it with [custom methods and variables](/code/builtin/overview.md) and [data transformation functions](/code/builtin/data-transformation-functions/index.md). These features make it easier to perform common tasks like getting data from other nodes or accessing workflow metadata.
 
-n8n supports two libraries:
+n8n additionally supports two libraries:
 
-- [Luxon](https://github.com/moment/luxon/){:target=_blank .external-link}, for working with data and time.
+- [Luxon](https://github.com/moment/luxon/){:target=_blank .external-link}, for working with dates and time.
```

Comment:
Ha, this is a funny thing to have persisted for so long.

Reply:
I was quite amazed!